src/helpers/formikUtils.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FieldArray } from 'formik';
+import { get } from 'lodash';
 
 export const getTouchErrorByField = (fieldName, { touched, errors }) => ({
-  touched: touched[fieldName],
-  error: errors[fieldName],
+  touched: get(touched, fieldName, ''),
+  error: get(errors, fieldName, ''),
```

Comment:
will not it fail if `fieldName` contains dot inside?

Reply:
it won't. that's why I use the lodash's `get` for that specific reason - reading dot notation